web/src/pages/projects/indiv.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx */
```

Comment:
This, along with `lent.tsx`, `oslo-jazz.tsx` and `tabetalt.tsx` should probably be created using a common component and by [using `gatsby-node.js`](https://www.gatsbyjs.com/docs/reference/routing/creating-routes/#using-gatsby-nodejs).